engine/launcher/src/main/scala/org/enso/launcher/releases/GraalCEReleaseProvider.scala

```diff
@@ -0,0 +1,65 @@
+package org.enso.launcher.releases
+
+import java.nio.file.Path
+
+import org.enso.cli.TaskProgress
+import org.enso.launcher.OS
+import org.enso.launcher.components.RuntimeVersion
+import org.enso.launcher.releases.github.GithubReleaseProvider
+
+import scala.util.{Failure, Success}
+
+/**
+  * [[RuntimeReleaseProvider]] implementation providing Graal Community Edition
+  * releases from the given [[ReleaseProvider]].
+  */
+class GraalCEReleaseProvider(releaseProvider: ReleaseProvider)
+    extends RuntimeReleaseProvider {
+  override def packageFileName(version: RuntimeVersion): String = {
```

Comment:
Doc?

Reply:
This is just an implementation of the `packageFileName` defined and documented in the base `RuntimeReleaseProvider`, should it have separate documentation?